src/main/features/claim/views/confirmation.njk

```diff
@@ -15,8 +15,11 @@
           <span>{{ t('Your claim number:') }}</span>
           <h1 class="bold-large">{{ claimNumber }}</h1>
         </div>
+        <p>
+          <span class="bold">{{ claim.respondedAt | date }}</span>
+        </p>
         <p class="receipt-download-container">
-          <a href="/claim/{{ externalId }}/receipt">{{ t('Download your claim as a PDF') }}</a>
+          <a href="/claim/{{ externalId }}/receipt">{{ t('Download your claim form') }}</a>
```

Comment:
@cidlockie Does features to be built need to be updated?
i.e. this content still has the old one:
http://hmcts-cmc-prototype.herokuapp.com/prototypes/features-to-be-built/claimant/confirmation-page